include/Ideen_und_Konzepte.tex

```diff
@@ -5,7 +5,9 @@ \chapter{Ideen und Konzepte}
 
 \section{Algorithmensuche}
 
-Um Diese Aufgabe zu lösen, wird ein bewährter und passender Algorithmus benötigt. Dazu wurde in der Recherchephase ermittelt, welche Arten von Algorithmen für solche Fälle verwendet werden. Die gefundenen Algorithmen, werden dann evaluiert und bewertet, sodass schlussendlich ein Algorithmus ausgewählt werden kann, der bestmöglich zur Lösung der Aufgabe verwendet werden kann. Die Alogrithmen wurden bewusst eher oberflächlich analysiert, um möglichst früh eine Art Algorithmus auswählen zu können. Danach wurde sich vertieft mit den ausgewählten Methoden auseinandergesetzt.\\
+Um Diese Aufgabe zu lösen, wird ein bewährter und passender Algorithmus benötigt. Dazu wurde in der Recherchephase ermittelt, welche Arten von Algorithmen für solche Fälle verwendet werden. Dabei wurden diverse Papers gelesen und deren Ergebnisse analysiert. Es wurde darauf geachtet wie gut die gefundenen Papers zu diesem Projekt passen, welche Ähnlichkeiten und Unterschiede es gibt. Ein weiterer wichtiger Punkt war die allgemeine Relevanz der Arbeiten, wie in Kapitel \ref{ch:StandDerTechnik} bereits besprochen wurden
+, dass gewisse Ansätze veraltet sind.
```

Comment:
why linebreak?

Reply:
there is no linebreak in this section ^^'